packages/buidler-core/test/internal/buidler-evm/provider/modules/evm.ts

```diff
@@ -411,56 +411,6 @@ describe("Evm module", function() {
 
         assert.equal(balanceAfterRevert, balanceBeforeTx);
       });
-
-      it("Should restore block filters", async function() {
```

Comment:
We don't revert filter ids anymore, which is nice. But I think we should test a few things:

- Tests created after the checkpoint we are reverting to shouldn't be present anymore. Or should they? Would love to know your opinion here.
- The next id isn't reverted.
- The filters' status gets reverted. Meaning that `eth_getFilterChanges` moves backward and re-returns some things.

Reply:
Update: most of these things are tested in `eth.ts`, right? Anything missing?

Reply:
My opinion is that all block hashes should be present since that way `eth_getFilterChanges` is consistent - it doesn't matter if `eth_getFilterChanges` was called before or after revert.